confluent_kafka/schema_registry/schema_registry_client.py

```diff
@@ -268,25 +378,22 @@ def delete_subject(self, subject):
             https://docs.confluent.io/current/schema-registry/develop/api.html#delete--subjects-(string-%20subject)
 
         """  # noqa: E501
-        self._rest_client.DELETE(url_formatter(SUBJECT, subject))
+        return self._rest_client.DELETE('subjects/' + subject)
 
     def get_version(self, subject_name, version='latest'):
         """
-        GET /subjects/(string: subject)/versions/(versionId: version)
-
-        Retrieves a specific schema registered to ``subject_name``.
+        Retrieves a specific schema registered under ``subject_name``.
 
         .. _note:
             "latest” returns the last registered schema under the specified
-            subject. Note that there may be a new latest schema that gets
-            registered right after this request is served.
+            subject.
 
         Args:
-            - subject_name (str): Subject name.
-            - version (int, optional): version number. Defaults to "latest".
+            subject_name (str): Subject name.
+            version (int, optional): version number. Defaults to "latest".
```

Comment:
"latest" is not an integer, maybe this should default to None instead as not to mix two distinct types for the same parameter, and there is no need to leak the SR protocol details here.